### PR TITLE
docs: add tutorial to debug KubeZoo binary locally

### PR DESCRIPTION
**Pull request by Silverglass**
#### What type of PR is this?

docs

#### What this PR does / why we need it:

add tutorial to debug KubeZoo binary locally

### CODE REVIEW

File: docs/debug-kubezoo-locally.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+### Launch `etcd` locally
+```console
+$ etcd
+{"level":"info","ts":"2021-09-17T09:19:32.783-0400","caller":"etcdmain/etcd.go:72","msg":... }
+⋮
+```
+
+### Run `KubeZoo` with printed parameters
+```console
+kubezoo --allow-privileged=true
```
Comment (SOF3) [inline docs/debug-kubezoo-locally.md:58]: Also add this to makefile?

File: hack/lib/gen_pki.sh [hunk 2]
```diff
@@ -16,7 +16,9 @@
 
 set -xue
 
-readonly TEMP_DIR=$(mktemp -d)
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
+KIND_SERVER="$(yq eval '.clusters.[]|select(.name=="kind-kind")|.cluster.server' ~/.kube/config)"
```
Comment (SOF3) [inline hack/lib/gen_pki.sh]: `kubectl config current-context` to support differently named contexts like below

Comment (Silverglass) [reply hack/lib/gen_pki.sh]: updated!~

**Review by SOF3 — APPROVED** [review]
:+1: fully reproducible on my local environment

**Pull request merged**